build.gradle

```diff
@@ -242,7 +242,7 @@ project('spring-rabbit') {
 
 		compile ("org.apache.logging.log4j:log4j-core:$log4j2Version", optional)
 
-		testCompile "commons-cli:commons-cli:$commonsCliVersion"
+//		testCompile "commons-cli:commons-cli:$commonsCliVersion"
 
```

Comment:
I have no recollection as to why I added it - it's only a test dependency; let's just remove it, not just comment it out.